### PR TITLE
Added a plain string example to the README

### PR DESCRIPTION
**Pull request by jolyonruss**
I thought it might be useful for people who aren't fans of Regex to see an example with plain strings.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +21,16 @@ gulp.task('templates', function(){
     .pipe(gulp.dest('build/file.txt'));
 });
 ```
+For strings
+```javascript
+var replace = require('gulp-replace');
+
+gulp.task('templates', function(){
+  gulp.src(['file.txt'])
+    .pipe(replace('{3}', 'foo'))
```
Comment (lazd) [inline README.md]: A simpler string would be nice -- folks may still confuse `{`/`}` for regex!

File: README.md [hunk 2]
```diff
@@ -20,6 +21,16 @@ gulp.task('templates', function(){
     .pipe(gulp.dest('build/file.txt'));
 });
 ```
+For strings
```
Comment (lazd) [inline README.md]: Please use a heading here:

```md
### Regex Replace
```

File: README.md [hunk 3]
```diff
@@ -11,6 +11,7 @@ npm install --save-dev gulp-replace
 
 Then, add it to your `gulpfile.js`:
 
+For Regex
```
Comment (lazd) [inline README.md]: Please use a heading here:

```md
### Regex Replace
```

**Comment by jolyonruss** [comment]
Good points, added another commit

**Comment by lazd** [comment]
Yay!

**Pull request merged**